x-pack/plugins/security_solution/cypress/e2e/detection_rules/rule_snoozing.cy.ts

```diff
@@ -165,23 +164,20 @@ describe('rule snoozing', () => {
     });
   });
 
-  // SKIPPED: https://github.com/elastic/kibana/issues/159349
-  describe.skip('Rule editing page / actions tab', () => {
+  describe('Rule editing page / actions tab', () => {
     beforeEach(() => {
       deleteConnectors();
     });
 
     it('adds an action to a snoozed rule', () => {
       createSnoozedRule(getNewRule({ name: 'Snoozed rule' })).then(({ body: rule }) => {
         visitWithoutDateRange(ruleEditUrl(rule.id));
-        // Wait for rule data being loaded
-        cy.get(RULE_INDICES).should('be.visible');
         goToActionsStepTab();
 
         addEmailConnectorAndRuleAction('abc@example.com', 'Test action');
         saveEditedRule();
 
-        goToRuleEditSettings();
```

Comment:
If I've understood correctly, the problem was with Cypress navigating to the rule edit page too quickly. However, I'm not entirely clear on how changing the navigation method resolves that issue. @maximpn, could you please elaborate?

Reply:
There is something going on under the hood. I've tried to reproduce the problem manually but without success. Most probably the root cause that ES doesn't return an updated rule instantly or it's an UI problem though I haven't found any clear confirmation.

In the other words full page loading just takes more time or it overcomes an UI problem. Anyway the test shouldn't be sensitive to the implementation details.

I've ran Flaky Test Runner to make sure this test doesn't fail. You can find the build [here](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/2459).